src/app/data-entries/select/select-input-directives/select-input-directives.html

```diff
@@ -0,0 +1,14 @@
+<div class="select-wrapper">
+    <igx-select [(ngModel)]="selected">
+        <label igxLabel>Pick a fruit</label>
+        <igx-prefix>
+            <igx-icon color="green" style="width:48px; height:32px;">BIO</igx-icon>
```

Comment:
Ooops, this shouldn't really be an igx-icon:
```suggestion
            <span class="bio-prefix">BIO</span>
```
And then you'd need an extra style (see below). This will obv need to be reflected in the docs snippet.